using-mobile-extensions/mobile-core/signal/README.md

```diff
@@ -0,0 +1,211 @@
+
+By leveraging the same triggers and traits that you use to display an in-app message, you can configure the SDK to send customized data to a third-party destination. Also, with the appropriate permissions and configurations, you can use the `collectPii` API to send PII to a remote server.
+
+Here are the implications of the privacy status for this extension:
+
+- When the `global.privacy` configuration is set to `optedout`, the Signals extension clears all queued hits and drops future network send requests.
+- If the configuration is `optunknown`, the network sends requests that are queued but not sent out.
+- If the configuration is `optedin`, the network requests to a third-party destination are allowed, provided the trigger conditions match.
+
+To get started with Identity, complete the following steps:
+
+1. Add the **Signal** framework to your app.
+2. Define the necessary rules in Adobe Experience Platform Launch. 
+3. Implement the SDK APIs to send PII data to external destinations.
+
+## Add the Signal framework to your app
+
+{% tabs %}
+{% tab title="Android" %}
+
+#### Java
+
+Add the  [Mobile Core](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core) extension to your project using the app's Gradle file:
+
+```
+implementation 'com.adobe.marketing.mobile:sdk-core:1.+'
+```
+
+Import the library:
+
+```java
+import com.adobe.marketing.mobile.*;
+```
+
+{% endtab %}
+
+{% tab title="iOS" %}
+
+#### iOS
+
+1. Add the  [Mobile Core](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core) extension to your project using Cocoapods:
+
+![](./img/signal_config_objc.png)
+
+
+
+2. Import the Signal extension:
+
+**Objective-C**
+
+```objectivec
+#import  "ACPSignal.h"
+```
+
+**Swift**
+
+In swift, the ACPCore includes ACPSignal:
+
+```swift
+import ACPCore
+```
+
+{% endtab %}
+
+{% tab title="React Native" %}
+
+Import the Signal extension:
+
+#### JavaScript
+
+```jsx
+import {ACPSignal} from '@adobe/react-native-acpcore';
+```
+
+{% endtab %}
+{% endtabs %}
+
+## Register the Signal extension
+
+The `registerExtension()` API registers the Signal extension with the MobileCore extension. This API allows the extension to send and receive events to and from the Mobile SDK.
+
+To register the Signal extension, use the following code sample:
+
+{% tabs %}
+{% tab title="Android" %}
+After calling the `setApplication()` method in the `onCreate()` method, register the extension. If the registration was not successful, an `InvalidInitException` is thrown.
+
+#### Java
+
+```java
+public class MobiletApp extends Application {
+@Override
+public void onCreate() {
+super.onCreate();
+     MobileCore.setApplication(this);
+     try {
+         ...........................
```

Comment:
Let's also include configureWithAppId and MobileCore.start. Analytics is a good example here https://aep-sdks.gitbook.io/docs/using-mobile-extensions/adobe-analytics#register-analytics-with-mobile-core